lib/serviceClient.ts

```diff
@@ -382,7 +361,7 @@ function createDefaultRequestPolicyFactories(credentials: ServiceClientCredentia
   }
 
   if (utils.isNode) {
```

Comment:
Remove this check now that our UserAgentPolicy works with both Node.js and browsers.